StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -454,8 +454,21 @@ extension PaymentSheet {
             case .withPaymentMethod(let paymentMethod):
                 confirmWithPaymentMethod(paymentMethod, nil, false)
             case .withPaymentDetails(let linkAccount, let paymentDetails):
-                // shouldSave is false, as we don't show a save checkbox in the Link VC
-                confirmWithPaymentDetails(linkAccount, paymentDetails, paymentDetails.cvc, false)
+                let shouldSave = false // always false, as we don't show a save-to-merchant checkbox in Link VC
+
+                if elementsSession.linkPassthroughModeEnabled {
+                    linkAccount.sharePaymentDetails(id: paymentDetails.stripeID, cvc: paymentDetails.cvc) { result in
+                        switch result {
```

Comment:
Seems like we should add a test to validate that these functions are being called when `linkPassthroughModeEnabled` == true.